tests/unit/introspection-2.tcl

```diff
@@ -26,6 +32,20 @@ start_server {tags {"introspection"}} {
         assert {[r object idletime foo] < 2}
     }
 
+    test {Operations in no-touch mode do not alter the last access time of a key} {
+        r set foo bar
+        r client no-touch on
+        set oldlru [getlru foo]
+        after 1000
```

Comment:
now that our resolution is milliseconds, and is no longer relative to the current time, we can reduce this sleep to 1 or 10 ms, no need to wait a full second, right?

Reply:
But I found in server.h:
```
#define LRU_CLOCK_RESOLUTION 1000 /* LRU clock resolution in ms */
```
And `debug object` command does return LRU in seconds.
Should this macro be changed?

Reply:
Ohh sorry. I mixed this up with TTL.
So sleeping 1 second guarantees that the lru would have been different if touch wasn't disabled.. ok